test/applications/MortgageApplication/application-conf/Cobol.properties

```diff
@@ -59,6 +64,12 @@ cobol_linkDebugExit=    INCLUDE OBJECT(@{member})  \n    INCLUDE SYSLIB(EQAD3CXT
 # can be overridden by file properties
 cobol_linkEdit=true
 
+#
+# store abbrev git hash in ssi field
+# available for buildTypes impactBuild, mergeBuild and fullBuild 
+# can be overridden by file properties
+cobol_storeSSI=true 
```

Comment:
Same question here. I did not make these changes on my branch - are they intentional?

Reply:
@lburgess07 Noticed that the changes are in test/applications/... ?